src/components/FeatureViews/Scheduler/SessionFilters.js

```diff
@@ -51,9 +51,7 @@ function SessionFilters({ onInstructorSelect, InstructorValue, InstructorOptions
             className={"session-filter"}
             onClose={handleClick}
         >
-            <MenuItem
-                disabled={true}
-            > Select Filter</MenuItem>
+            <MenuItem selected={true}> Select Filter</MenuItem>
```

Comment:
If a prop is equal to the boolean `true`, the `={true}` can be omitted (it defaults to the value `true`):

    <MenuItem selected>...